spec/views/users/search.html.erb_spec.rb

```diff
@@ -2,15 +2,23 @@
 
 RSpec.describe 'users/search', type: :view do
   let(:users) { FactoryBot.create_list(:user, 3) }
+  let(:contacts) { FactoryBot.create_list(:user, 3) }
+  let(:user1) { FactoryBot.create :user }
 
   before do
-    assign(:users, users)
+    assign(:users, contacts)
     assign(:users_to_add, users)
-    sign_in users.first
+    sign_in user1
     render
   end
 
-  it 'renders a list of users before searching' do
+  it 'renders a list of contacts' do
```

Comment:
What's your thought behind the changes in this test? You have renamed the variable in the test and added the existing test again with the renamed variable but that shouldn't change the behaviour of the test. I would therefore restore this file to its current state on dev

Reply:
We have *users* and *users_to_add* and we want to check that both groups are being rendered. That's why we have two test cases.

Reply:
That's a good idea, but that is not what the test does :sweat_smile:

Reply:
`assign(:users, contacts)` is like executing `@users = contacts` in the controller before rendering the view (    assign(:users, contacts))

Reply:
I just changed the code back to the dev status.